### DISCUSSION TITLE
Seems hard to get SigninPage checkbox `remember` value?

### DISCUSSION THREAD

**Discussion by zhang1career** [mui/toolpad#4403] [Discussions]
Hi, I'm a new js/ts developer, struggling to get a checkbox's `remember` value from the SigninPage in the package `toolpad-core` v0.8.1.

The only way I know to get the value is by Checkbox's onClick handler, but there seems no one exposured to user in SignInPage.

Would you be so kind to offer some advice?

**Comment by bharatkashyap** [comment]
Hi! Thanks for reporting this - this is a bug in the `SignInPage`  because of a missing `name` attribute. Once fixed, you should be able to get the value from the `formData` param of the `signIn` function using `formData.get("remember")`
